### PR TITLE
Pillar of Spring 2023 Expansion Part 2: kotahi edition

### PR DESCRIPTION
**Pull request by novaepee**
## About The Pull Request

Expand starboard of PoS!

image

Add more lights in medbay.

Update NOT UNO card sprites!

image

Allow marines to spawn in with NOT UNO cards!

Vend out NOT UNO decks from the Utility tab in the Armament vendor!

## Why It's Good For The Game

https://user-images.githubusercontent.com/6610922/211707692-bcf8b3d6-a7f1-4777-9b33-78982eae0eae.mp4

Since I have expanded portside of PoS for medbay and trauma bay, I ought to do the same for starboard. With the INCREASED AWARENESS OF NOT UNO, I want this PR to increase accessability of NOT UNO.

We have NOT UNO for 3 years from this PR #4387, and yet UNTIL TODAY, we saw its usage. We need more of it!

## Changelog

:cl:
add: Expand starboard of PoS!
Add: more lights in medbay.
imageadd: Update NOT UNO card sprites!
add: Allow marines to spawn in with NOT UNO cards!
/:cl:

### CODE REVIEW

File: code/game/objects/items/toys/cards.dm [hunk 1]
```diff
@@ -315,8 +315,8 @@
 
 
 /obj/item/toy/deck/kotahi
-	name = "KOTAHI deck"
-	desc = "A flashy deck of Nanotrasen KOTAHI playing cards. Usually sold alongside crayon packages."
+	name = "KOTAHI deck (NOT UNO)"
+	desc = "A flashy deck of Nanotrasen KOTAHI playing cards that are definitely NOT UNO. Usually sold alongside crayon packages."
```
Comment (TiviPlus) [inline code/game/objects/items/toys/cards.dm:319]: revert

File: code/modules/client/preferences_gear.dm [hunk 2]
```diff
@@ -219,3 +219,10 @@ GLOBAL_LIST_EMPTY(gear_datums)
 	path = /obj/item/clothing/glasses/mgoggles/prescription
 	cost = 2
 	slot = SLOT_GLASSES
+
+/datum/gear/kotahi
+	display_name = "kotahi (NOT UNO)"
```
Comment (TiviPlus) [inline code/modules/client/preferences_gear.dm:224]:
```suggestion
	display_name = "Kotahi deck"
```

**Review by TiviPlus — CHANGES_REQUESTED** [review]
Free with your xbox

**Review by RipGrayson — CHANGES_REQUESTED** [review]
Missing a wall here
van16
These AI nodes aren't connected
van17

**Pull request merged**